.github/workflows/gempush.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Gem
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
```

Comment:
```suggestion
    - uses: actions/checkout@v2
```

Reply:
This is better to "control" what we use, I need to change this is all the other repos.